core/src/test/java/io/atomix/core/AtomixRule.java

```diff
@@ -0,0 +1,144 @@
+package io.atomix.core;
```

Comment:
We don't add a license header any more?

Reply:
Was not intended I just have no configuration for atomix :man_shrugging: Personally I would add then apache with camunda mentioning.